src/main/java/MysqlUserDAO.java

```diff
@@ -20,22 +22,50 @@ public void getUsers() {
 
 	public User getUser(String userName) {
 
-		MysqlDAO mysql = new MysqlUserDAO();
-
 		String sql_query = "SELECT * FROM users WHERE userName=\"" + userName + "\"";
-		HashMap<String, Object> row = mysql.getQuery(sql_query).get(0);
+		HashMap<String, Object> row = this.getQuery(sql_query).get(0);
 
 		// TODO: Create the actual user object like a man
 		return new User();
 	}
 
+	public Boolean addUser(String userName) {
+
+		String sql_query = String.format("INSERT INTO %s (username) values (\"%s\")",
+						TABLE_NAME, userName);
+
+		// Returns true if query is valid and username is unique, otherwise false
+		return this.updateQuery(sql_query);
+	}
+
+	public Boolean deleteUser(String userName) {
+
+		String sql_query = String.format("DELETE FROM %s WHERE username = \"%s\"",
+						TABLE_NAME, userName);
+
+		// Returns true if query is valid and username exists
+		return this.updateQuery(sql_query);
+	}
+
 	public static void main(String[] args) {
```

Comment:
For future reference, the preferred way to run these types of test is probably through JUnit so that it can integrate with the build.